app/models/manageiq/providers/vmware/infra_manager/inventory/saver.rb

```diff
@@ -1,11 +1,11 @@
 class ManageIQ::Providers::Vmware::InfraManager::Inventory::Saver
   include Vmdb::Logging
 
-  def initialize
+  def initialize(threaded: true)
     @join_limit  = 30
     @queue       = Queue.new
     @should_exit = Concurrent::AtomicBoolean.new
-    @threaded    = ENV["RAILS_ENV"] != "test"
```

Comment:
This was kind of a hacky workaround for the saver thread being created in the collector and not having the ability to set it to unthreaded in specs.